dango/types/src/lending/market.rs

```diff
@@ -65,49 +74,40 @@ impl Market {
         // This can happen if 100% of the supply is borrowed, which can then cause
         // borrowing to outgrow the supply due to interest accrual.
         if utilization_rate > Udec128::new_percent(100) {
-            return Ok(Udec128::new_percent(100));
+            return Ok(Bounded::new_unchecked(Udec128::new_percent(100)));
         }
 
-        Ok(utilization_rate)
+        Ok(Bounded::new_unchecked(utilization_rate))
```

Comment:
What's the point of using Bounded if you instantiate with unchecked?

Reply:
If a function returns a `Bounded`, if conveys to the caller the idea that the function guarantees the result is bounded.

Although in the actual implementation of the function, we already check the value of `utilization_rate` against `0` and `1`, no need to check again, so I do `Bounded::new_unchecked`.